### PR TITLE
Remove code for the help and services around the world page

### PR DESCRIPTION
**Pull request by JonathanHallam**
Whitehall no longer renders this page so the code can be removed

Trello - https://trello.com/c/9zmuoJrZ/519-remove-code-for-help-and-services-around-the-world-from-whitehall

⚠️ This repo is Continuously Deployed: make sure you [follow the guidance](https://docs.publishing.service.gov.uk/manual/development-pipeline.html#merge-your-own-pull-request) ⚠️

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,12 +5,6 @@ Whitehall is deployed in two modes:
 - 'admin' for publishers to create and manage content (e.g. <http://whitehall-admin.dev.gov.uk/government/admin/news/new>)
 - 'frontend' for rendering content to the public that has yet to be migrated to using Publishing API (see live examples that follow)
```
Comment (brucebolt) [inline README.md:6]: Could this line be updated to say `'frontend' is a legacy mode that was used for rendering content to the public that had yet to be migrated to using Publishing API - work to remove this is being carried out by the Publishing Platform team`?

File: app/views/world_locations/index.html.erb [hunk 2]
```diff
@@ -1,67 +0,0 @@
-<% page_title sanitize(t("world_location.content.heading").gsub!(/&nbsp;/, " ")) %>
-<% page_class "govuk-width-container world-locations-index index-list-page" %>
-
-<div class="govuk-grid-row">
-  <div class="govuk-grid-column-two-thirds">
-    <header class="js-filter-list">
-      <%= render "govuk_publishing_components/components/title", {
-        title: sanitize(t("world_location.content.heading"))
```
Comment (brucebolt) [inline app/views/world_locations/index.html.erb:8]: Can these locales be removed from the locale files now?  It'll save a bit of work when we do the tidy-up cards.

**Review by brucebolt — COMMENTED** [review]
Looks good.  I've left a few comments and notice there's also a missing `d` in the word `around` in the commit message.

File: app/assets/javascripts/frontend/on_ready.js [hunk 3]
```diff
@@ -2,7 +2,6 @@
 jQuery(function ($) {
   $('.govspeak').enhanceYoutubeVideoLinks()
 
-  GOVUK.worldLocationFilter.init()
   GOVUK.hideDepartmentChildren.init()
   GOVUK.filterListItems.init()
```
Comment (jkempster34) [inline app/assets/javascripts/frontend/on_ready.js:6]: I think `filterListItems` is only used on the world index page, so that can go too

Comment (JonathanHallam) [reply app/assets/javascripts/frontend/on_ready.js]: cheers :)

File: README.md (hunk 1 above)
Comment (JonathanHallam) [reply README.md]: 👍

**Review by brucebolt — APPROVED** [review]

**Comment by JonathanHallam** [comment]
> There's a failing test. Looks like `test/functional/world_locations_controller_test.rb` also needs to be removed.

yeah, I think this must have been reintroduced in one of my many rebases, had everything green this morning.

**Pull request merged**